models/dao/user-survey.dao.js

```diff
@@ -52,6 +52,19 @@ const comparators = {
         }
         return false;
     },
+    'in-zip-range': function (answers, ruleAnswers) {
+        const answersValues = answers.map(answer => answer.value);
+        let found = false;
+        for (let i = 0; i < ruleAnswers.length; i += 1) {
+            const inZipRangeValues = ruleAnswers[i].meta.inRangeValue;
```

Comment:
For this one, I've used the explicit for-loop because that's the only way I know how to use `break` in JavaScript. Is there a better solution such that I don't have to rely breaking out of a loop?

Reply:
You are already using "some" inside loop, why don't you use it outside as well?